pkg/common/consumer/consumer_handler.go

```diff
@@ -108,24 +120,50 @@ func (consumer *SaramaConsumerHandler) Cleanup(session sarama.ConsumerGroupSessi
 func (consumer *SaramaConsumerHandler) ConsumeClaim(session sarama.ConsumerGroupSession, claim sarama.ConsumerGroupClaim) error {
 	consumer.logger.Infow(fmt.Sprintf("Starting partition consumer, topic: %s, partition: %d, initialOffset: %d", claim.Topic(), claim.Partition(), claim.InitialOffset()), zap.String("ConsumeGroup", consumer.handler.GetConsumerGroup()))
 	consumer.handler.SetReady(claim.Partition(), true)
+	c := make(chan bool)
```

Comment:
`c` as in `channel` ?

Reply:
yep

Reply:
@devguyio?  :-)

Reply:
just my 2 cents (feel free to ignore : ) but I tend to agree with @matzew and try to avoid these short variable names ("c", "ce", "hctx", "f", "sch", etc).   It just requires more mental energy when reading the code to remember what they refer to rather than if they were spelled out (stopChan, checkedEntry, etc...).